### DISCUSSION TITLE
Android GitHub app

### DISCUSSION THREAD

**Discussion by ktauchathuranga** [community/community#127497] [Mobile]
GitHub app in Android is not correctly rendering graphs.

![Screenshot_20240606_103854_GitHub.jpg](https://github.com/community/community/assets/79928743/73770ff9-c614-4122-88e4-5ae25e758757)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ktauchathuranga ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by ktauchathuranga** [reply]
oh thank youu... i am sorry for the oversight,
😇

**Comment by elisealix22** [comment]
@ktauchathuranga thank you so much for reporting this! We are tracking this internally and I added your feedback to our open issue for us to look into. Please let us know if you have any other requests in the meantime!
